### PR TITLE
Solved Error NoReverseMatch

### PR DESCRIPTION
**Pull request by SurajDadral**

### CODE REVIEW

File: src/librehatti/settings.py [hunk 1]
```diff
@@ -53,13 +53,7 @@
...
 MEDIA_URL = ''
 
-STATIC_ROOT = ''
-
-STATIC_URL = '/librehatti/static/'
-
-STATICFILES_DIRS = (
-    '../../static',
-)
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')                                                  STATIC_URL = '/librehatti/static/'                                                              STATICFILES_DIRS = [                                    os.path.join(BASE_DIR, '../static')     ]
```
Comment (amrit3701) [inline src/librehatti/settings.py:56]: Looks like `STATIC_ROOT`, `STATIC_URL` and `STATICFILES_DIRS` are all in one line. Separate it.

File: src/templates/voucher/sessionselect.html [hunk 2]
```diff
@@ -1,6 +1,10 @@
 {% extends "base.html" %}
 {% block content %}
-<form action='{% url 'librehatti.suspense.views.sessionselect' %}' method='POST'>
+
+# modified <form action='{% url 'librehatti.suspense.views.sessionselect' %}' method='POST'> to make compatible with django2.0
```
Comment (amrit3701) [inline src/templates/voucher/sessionselect.html:4]: Why the above line start with `#`?

File: src/librehatti/settings.py (hunk 1 above)
Comment (SurajDadral) [reply src/librehatti/settings.py:56]: Corrected in https://github.com/GreatDevelopers/LibreHatti/pull/524

File: src/templates/voucher/sessionselect.html (hunk 2 above)
Comment (SurajDadral) [reply src/templates/voucher/sessionselect.html:4]: I mixed comments syntax for HTML and Python.
But corrected in https://github.com/GreatDevelopers/LibreHatti/pull/524

**Pull request merged**